File: 01 代码包/后端/src/main/java/com/smartshoppingvalley/Admin/entity/Shop.java
package com.smartshoppingvalley.Admin.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

import java.io.Serializable;
import java.sql.Time;

/**
 * <p>
 *
 * </p>
 */
@Data
@EqualsAndHashCode(callSuper = false)
@Accessors(chain = true)
public class Shop implements Serializable {

    private static final long serialVersionUID = 1L;

    /**
     * 主键
     */
    @TableId(value = "id", type = IdType.AUTO)
    private Long id;

    /**
     * 店铺名称
     */
    private String name;

    /**
     * logo
     */
    private String avatar;

    /**
     * 店铺介绍
     */
    private String description;

    /**
     * 开始营业时间
     */
    private Time openingTime;

    /**
     * 结束营业时间
     */
    private Time closingTime;

    /**
     * 详细地址
     */
    private String address;

    /**
     * 联系信息
     */
    private String phone;


}

File: 01 代码包/管理端/platform-side - 副本/src/store/Shop.js
import { getShopApi, changeShopApi } from '../api';

export default {
    namespaced: true,
    state: {
        shop: {
            // id: 0,
            // name: '乐淘',
            // avatar: '',
            // description: '商品丰富，欢迎选购',
            // openingTime: '07:00',
            // closingTime: '24:00',
            // address: '江苏省南京市江宁区东南大学',
            // phone: '18456325410'
        }
    },
    actions: {
        getShop(context, data) {
            return getShopApi()
                .then(({ data }) => {
                    // console.log('shop');
                    // console.log(data);
                    context.commit('GET_SHOP', data.data);
                })
        },
        changeShop(context, data) {
            console.log(data);
            return changeShopApi(data);
        },
        change_Time_1(context, value) {
            context.commit('CHANGE_TIME_1', value);
        },
        change_Time_2(context, value) {
            context.commit('CHANGE_TIME_2', value);
        },
    },
    mutations: {
        GET_SHOP(state, value) {
            state.shop = value;
        },
        CHANGE_TIME_1(state, value) {
            state.shop.openingTime = value;
        },
        CHANGE_TIME_2(state, value) {
            state.shop.closingTime = value;
        }
    }
}